app/controllers/api/v2/containers_controller.rb

```diff
@@ -111,17 +107,20 @@ def logs
                 :tail   => (params[:tail]   || 100)) }
       end
 
+      POWER_ACTIONS = %(start stop status)
+
       api :PUT, '/containers/:id/power', N_('Run power operation on a container')
       api :PUT, '/compute_resources/:compute_resource_id/containers/:id/power',
-          N_('Run power operation on a container in a compute resource')
+          N_('Run power operation on a container on a compute resource')
       param :id, :identifier, :required => true
       param :compute_resource_id, :identifier
-      param :power_action, String,
+      param :power_action, ContainersController::POWER_ACTIONS,
             :required => true,
-            :desc     => N_('power action, valid actions are (start), (stop), (status)')
+            :desc     => N_("power action, valid actions are") +
```

Comment:
This shouldn't be concatenated, you could use "valid actions are %s" but if it's in the second param then you shouldn't need it at all.  (We do this in core in a few places.)